int8 transceiverID
int8 responseID
string transceiverModelName
string data